website/docs/providers/rac/index.md

```diff
@@ -1,38 +1,56 @@
+
+When a user starts the RAC application, the app communicates with the authentik server, which then connects to an instance of the outpost (the exact instance is selected dynamically based on connection load). After the outpost is selected, then the authentik server sends the outpost the instructions (based on the data you defined in the endpoint) required to connect to the remote machine.
+
+After the connection to the remote machine is made, the outpost sends a message back to the authentik server (via websockets), and the web browser opens the websocket connection to the remote machine.
+
+### Endpoints
 
-Unlike other providers, where one provider-application pair must be created for each resource you wish to access, the RAC provider handles this slightly differently. For each machine (computer/server) that should be accessible, an _Endpoint_ object must be created within an RAC provider.
+Unlike other providers, where one provider-application pair must be created for each resource you wish to access, the RAC provider handles this slightly differently. For each remote machine (computer/server) that should be accessible, you create an _Endpoint_ object within a single RAC provider. (And as mentioned above, a single provider-application pair is used for all remote connections.)
 
-The _Endpoint_ object specifies the hostname/IP of the machine to connect to, as well as the protocol to use. Additionally it is possible to bind policies to _endpoint_ objects to restrict access. Users must have access to both the application the RAC Provider is using as well as the individual endpoint.
+The _Endpoint_ object specifies the hostname/IP of the machine to connect to, as well as the protocol to use. Additionally it is possible to bind policies to _endpoint_ objects to restrict access. Users must have access to both the application that the RAC Provider is using as well as the individual endpoint.
 
-Configuration like credentials can be specified through _settings_, which can be specified on different levels and are all merged together when connecting:
+Configuration such as credentials can be specified through _settings_, which can be specified on different levels and are all merged together when connecting:
 
 1. Provider settings
 2. Endpoint settings
 3. Connection settings (see [Connections](#connections))
 4. Provider property mapping settings
 5. Endpoint property mapping settings
 
-## Connections
+### Connections
 
-Each connection is authorized through the policies bound to the application and the endpoint, and additional verification can be done with the authorization flow.
+Each connection is authorized through the policies that are bound to the application and the endpoint, and additional verification can be done with the authorization flow.
```

Comment:
Suggestion?  "Each connection is authorized through authentik Policy objects bound to the application and the endpoint."